manifest.json

```diff
@@ -41,9 +41,24 @@
     "type": "object",
     "bindingBounded": true,
     "properties": {
-      "freeShippingAmount": {
-        "title": "Free Shipping Amount",
-        "description": "Enter the minimum amount for free shipping",
+      "freeShippingTradePolicy1": {
```

Comment:
Since this is a repeating property, let's use an array so we don't have to add more if a store wants different amounts for more than 4 trade policies.
Something like this should work:
```
.
.
.
"properties": {
      "freeShippingTradePolicies": {
        "title": "Trade Policies",
        "description": "Free shipping per trade policies",
        "type": "array",
        "items": {
          "type": "object",
          "properties": {
            "tradePolicy": {
              "title": "Trade Policy (Sales Channel)",
              "type": "string"
            },
            "freeShippingAmount": {
              "title": "Free Shipping Amount",
              "description": "Enter the minimum amount for free shipping on this trade policy",
              "type": "number"
            }
          }
        }
      }
    }
.
.
.
```